roles/ovirt-cluster-upgrade/README.md

```diff
@@ -0,0 +1,57 @@
+oVirt cluster upgrade
+=========
+
+This role will iterate through all hosts in cluster and upgrades them.
+
+Requirements
+------------
+
+ * Ansible version 2.3 or higher
+ * Python SDK version 4 or higher
+
+Role Variables
+--------------
+
+| Name               | Default value         |                            |
+|--------------------|-----------------------|----------------------------| 
+| cluster            | Default               | Name of the cluster to be upgraded |
+| low_prio_vms       | UNDEF                 | List of VM names to be stopped before upgrade |
```

Comment:
You may not cover all the VMs you want to shutdown by one pattern. I agree with `stopped_vms`.